portal-frontend/src/app/features/notice-of-intents/edit-submission/additional-information/add-structure-dialog/add-structure-dialog.component.scss

```diff
@@ -0,0 +1,132 @@
+@use '../../../../../../styles/functions' as *;
+@use '../../../../../../styles/colors';
+
+.actions {
+  display: flex;
+  padding: rem(16) rem(24);
+  flex-direction: column-reverse;
+  gap: rem(8);
+
+  @media screen and (min-width: $tabletBreakpoint) {
+    flex-direction: unset;
+    gap: unset;
+    margin-top: unset;
+  }
+}
+
+.btn-group {
+  width: 100%;
+  display: flex;
+  gap: rem(16);
+
+  button {
+    flex: 1 1 auto;
+  }
+
+  @media screen and (min-width: $tabletBreakpoint) {
+    width: unset;
+
+    button {
+      flex: unset;
+    }
+  }
+}
+
+:host::ng-deep {
+  .warning-banner {
+    padding: 0.5rem !important;
+  }
+}
+
+.mat-button-toggle-checked.mat-button-toggle-appearance-standard {
+  background-color: colors.$primary-color-light;
+}
+
+:host::ng-deep {
+  .field-error {
+    color: colors.$error-color;
+    font-size: rem(15);
+    font-weight: 700;
+    display: flex;
+    align-items: center;
+    margin-top: rem(4);
+  }
+}
+
+.delete-btn {
+  order: -1;
+  margin-right: auto !important;
+  width: 100%;
+
+  @media screen and (min-width: $tabletBreakpoint) {
+    width: unset;
+  }
+}
+
+.warning-icon {
+  padding-right: 27px;
```

Comment:
And please replace all `px` values with `rem` function as well, e.g., `27px` -> `rem(27)`.